appveyor.yml

```diff
@@ -0,0 +1,19 @@
+version: '{build}'
+image: Visual Studio 2019
+build_script:
+- pwsh: >-
+    mkdir build
+
+    cd build
+
+    cmake .. -DDBG_MACRO_BUILD_TESTS=ON -DDBG_MACRO_BUILD_EXAMPLES=ON
+
+    msbuild dbg_macro.sln
+
+    ctest -C Debug
```

Comment:
Can you please use
```
ctest --output-on-failure -C Debug
```
so we can see the error message in the log?

Reply:
Sure!

Reply:
Done. Now we have to wait for it to build.
https://ci.appveyor.com/project/xorz57/dbg-macro/builds/25475128